course/models.py

```diff
@@ -58,3 +61,12 @@ class Enrollment(models.Model):
     # Timestamps
     updated_at = models.DateTimeField(auto_now=True)
     created_at = models.DateTimeField(auto_now_add=True)
+
+
+class Session(models.Model):
+    start_date_time = models.DateTimeField()
+    end_date_time = models.DateTimeField()
+    price = models.DecimalField(max_digits=6, decimal_places=2, validators=[MinValueValidator(Decimal('0.00'))])
```

Comment:
price should be associated with the course, not the individual session. Also, the session model has already been added in the scheduler app

Reply:
rebase onto master to merge